corehq/apps/app_manager/models.py

```diff
@@ -5579,6 +5581,17 @@ def get_latest_master_release(self):
         else:
             raise ActionNotPermitted
 
+    @classmethod
+    def wrap(cls, data):
+        self = super(LinkedApplication, cls).wrap(data)
```

Comment:
using `self` as a local variable in this method is a little confusing but I see we do it elsewhere in the file, so if you prefer it, it seems fine to keep.

Most of the lazy migrations we do seem to modify `data`, then return `super` on the modified dict, instead of modifying the returned object. Is there a reason to prefer one or the other here?

Reply:
fe44da88d5d